contents/docs/libraries/android/index.mdx

```diff
@@ -81,6 +87,38 @@ PostHog.capture(
 
 `userPropertiesSetOnce` works just like `userProperties`, except that it will **only set the property if the user doesn't already have that property set**.
 
+## Super Properties
```

Comment:
Is there any way to reset the super properties? I'm assuming `posthog.reset()` does that... might be worth mentioning

Reply:
Lol ignore, just read the next section